df sdfasdf

```diff
@@ -0,0 +1,23 @@
+[33m90c4d2e[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;33mtag: A-JavaDoc[m[33m, [m[1;31morigin/master[m[33m, [m[1;31morigin/HEAD[m[33m)[m master@{0}: merge branch-A-JavaDoc: Merge made by the 'recursive' strategy.
```

Comment:
What is this file for? Please avoid committing such files to github. You can store them somewhere else or gitignore it.